### ISSUE TITLE
CUH1216 support :)?

### ISSUE THREAD

**Issue by gotwig** [valentinbreiz/PS4-Linux-Loader#3]
I would like to ask for CUH-1216 support in the new versions :) 

Thank you for your work!

**Comment by valentinbreiz** [comment]
Hello @gotwig, the loader works with all models (I hope). The only problem is that there is a problem for PS4 Slim, Pro and new FATs (like CUH-1216...) in the Linux port of @fail0verflow.

If you want to know more about this problem: https://gbatemp.net/threads/ps4-linux-on-slim.494403/

**Comment by gotwig** [comment]
Hm the progress seems to be stopped over there... A display adapter could be an option, but is a bit messy

**Comment by shim80** [comment]
would it be possible to make Panasonic MN864729 drivers via reverse enginering ?

**Comment by valentinbreiz** [comment]
Yes @shim80, with a kernel dump we can look for the rights icc queries and study how MN864729 works.

**Comment by shim80** [comment]
What's new? Can I help?

**Comment by Astromatik** [comment]
Bonjour Valentin je suis aussi sur ce modèle et je attend avec impatience linux sur celui ci pour tester starbound steam. 
Si je peux aider sa sera avec plaisir. A bientôt

**Comment by valentinbreiz** [comment]
Hello @Astromatik, @shim80 and @gotwig. I can confirm that the work on MN864729 is done and working, eeply posted a working bzimage. For CUH-2016A it's not released but I tested and it works -release soon-!

**Comment by shim80** [comment]
Hi @valentinbreiz , I've downloaded the the new bzimage and tested it but Linux don't work for me (cuh-2015B). The monitor don't receive any signal after launch payload.

Also, some testers says there is not driver gpu on this bzimage...

**Comment by valentinbreiz** [comment]
2015B and 2016A have the same problem. It will be released soon.

**Comment by Astromatik** [comment]
Genial. Je te avoue que je n'ai pas tester car je n'ai aucune retomber sur ce modèle.
Je le fait cette après-midi pour voir si sa blackscreen.  
Crois tu qu'il faut plutôt utiliser fedora ou le nouveau Manjaro de logic sunrise ?

**Comment by shim80** [comment]
Manjaro me semble plus adapté. 

C'est curieux, car d'après Tikilou, un crash/black screen/écran éteint est normal et qu'il faut une bonne dizaine d'essais en moyenne pour obtenir un résultat.
Cependant, j'ai dû refaire l'essai au moins 30 fois sans jamais voir apparaître le shell Linux...

**Comment by gotwig** [comment]
Love your progress guys, keep it up :D Cant wait for Steam on my CUH1216 ;) <3

**Comment by Astromatik** [comment]
J'essaie de sender ton playload valentin mais j'ai que des mémoires insuffisantes...

**Comment by shim80** [comment]
@Astromatik , tu utilise quel fichier initramfs?

**Comment by Astromatik** [comment]
Celui de Eeply à sa sortie...

**Comment by shim80** [comment]
Eeply a sorti un fichier bzimage, pas initramfs.

**Comment by Astromatik** [comment]
... Non excuse. J'ai dû le choper sur un site parmi la centaine qu' il y a dans mon historique de recherche PTDR. Depuis 2semai nes c'est la quête du Saint graal linux sur ps4 CUH1216A...

**Comment by shim80** [comment]
Oh, ça va venir, et très bientôt.
Eeply dit sur son twitter avoir développé un payload pour les écrans 720p... D'après ce que j'ai compris, ça permet de corriger les défauts de résolution même si j'ai du mal à en saisir l'intérêt (la résolution peut-être changer après l'exécution de Linux, non?).

**Comment by Astromatik** [comment]
Ouais et puis c est quand même mieux que rien je trouve ^^ j'espère vraiment Qu'il vont faire ses modèles... C'est les dernières PHAT^^

**Comment by shim80** [comment]
@valentinbreiz , after a lot of try to run Linux, I've attempted to run it with PS4 connected to the TV (and not from my PC monitor), and it work!!! But why??!
What is the difference between HDMI PC monitor and TV? They both using HDMI!

**Comment by Astromatik** [comment]
Wouaw super avancée. Peux tu me dire en bref et selon toi les plus importantes étapes de installation grosso merdo. Je ai trouver aucun tuto fiable ainsi que les fichier que tu as utilisé comme çà je retranscrit çà se soir. Merci encore

**Comment by shim80** [comment]
@Astromatik , j'ai utilisé les fichiers fournis par Tikilou, dans son tuto de logic-sunrise (voir ici: http://www.logic-sunrise.com/news-981270-ps4-manjaro-linux-ps4-v1-disponible.html ).
J'ai flashé le fichier img via rufus sur ma clé USB. Ça a donné deux partitions : celle avec linux (format ext4), et celle avec les fichiers d'exécution du noyau (format fat32, donc visible sous windows).
Je n'ai rien eu à changer à tout ça.
Après, il suffit juste de brancher la clé sur la PS4 et d'exécuter le payload (cette étape varie selon ton firmware, et ton exploit)

**Comment by Astromatik** [comment]
Bon... Y a du mieux mais c'est pas encore çà...
je me scotch un shutdown sans vérification du disque dur interne à chaque playload à chaque fois... XD on va y arriver ^^

**Comment by Astromatik** [comment]
Ok Y a du mieux je pense ce matin. une fois le playload sender (et une seul fois ^^-)
Je croix que j'ai enfin quelques chose.
Ma console est actuellement encore allumé, blackscreen (en la connectant sur une TV Full HD Phillips, là ou elle est branché d'habitude ainsi que sur un TV Samsung)
Elle n 'est pas éteinte mais la lumière en haut est bleu mais elle clignote rose sur le coté bouton ON OFF / lecteur BR.
c'est normal ?
est t'il en train de boot sur l'installation mais ne reconnais toujour pas le modèle PHAT CUH1216A ou j'ai oublié des trucs?

**Comment by valentinbreiz** [comment]
https://twitter.com/eeply/status/977670090974998529 ;)

**Comment by Astromatik** [comment]
Yes je test çà demain encore merci à vous ValentinBreiz et Eeply !!!

**Comment by shim80** [comment]
@valentinbreiz Linux still don't work with my PC monitor (TV only)... 
A bug with MN864729 drivers?

**Comment by valentinbreiz** [comment]
Quel est le modèle de ta PS4, et sa version? Le bug peut venir de la configuration de ta télé aussi, mets en 1080i et le hdcp en 1.4, pas en 2.2.

**Comment by shim80** [comment]
@valentinbreiz -> PS4 Slim CUH-2015A OFW 4.55.
connectée à ma TV (Changhong 20" - 1366x768 - 50hz) via HDMI -> Linux s'affiche bien, mais la résolution est trop haute, donc seule une partie de l’écran est visible. 
connectée à mon écran PC (HP Pavilion 22cw - 1920x1080 - 60hz) via HDMI -> L’écran ne détecte rien et fini par passer en mode veille tout seul.

Le problème existe du coté  de l’écran PC et non de la TV. Je ne vois pas comment modifier dessus des paramètres comme mettre en 1080i ou changer le hdcp...

**Comment by Astromatik** [comment]
Sa y est je ai de l' image mais je pense comme shime je suis sur ma TV Samsung et j' ai l' impression d' avoir une moitié d'écran. C'est bizarre ma TV Philips ne affiche rien du tout... Portant Ma Philips affiche normalement ma Snes mini CD en 720 tranquillou. Devons nous fixer les résolutions sur le PS4-Linux-Loader selon le moniteur utilisé Valentin ?

**Comment by Tikilou** [comment]
Hello, juste un mot pour préciser, je n'ai pas encore essayé la nouvelle release du noyau Linux d'Eeeply mais je vais essayer.
Sur ps4 pro 7016B, sur firmware 4.55, en tout cas, la vidéo que j'ai montré de boot de manjaro, j'ai eu au moins une dizaine de crashs+reboot/moniteur éteint.
Et le crash se faisait dès le chargement du loader.

**Comment by shim80** [comment]
@Tikilou Depuis que j'ai mon nouveau lanceur de payload, je n'ai plus jamais échoué au lancement de Linux.
Par contre, je suis incapable d'y voir quoi que ce soit depuis mon écran PC.

**Comment by Astromatik** [comment]
Salut tout le monde. 
A la ligne rescueshell # je dois faire quoi ? : P
Je dois passer pour un gros naze mais linux et moi c'est un peu nouveau ^^
Habituellement quand je prend le PC ou le Mac je m'y retrouve toujours mieux.

**Comment by Astromatik** [comment]
OK comme la manip fedora merci beaucoup tout le monde je espère que vous ce projet linux avec des drivers à jours. Super boulot

**Comment by Astromatik** [comment]
Continuerai escuse moi

**Comment by shim80** [comment]
Il s'agit du projet de PSXITA. D'apres leur twitter, ils ont prevu de mettre en place les pilotes du son et du gpu. Par contre, je n'ai pas vu de mention concernant la mise en place des pilotes MN864729

**Comment by shim80** [comment]
@valentinbreiz J'ai essayé de compiler le noyau Linux: https://github.com/Ps3itaTeam/ps4-linux , mais sans succès... Est-ce que tu peux essayer de le compiler?

**Comment by Astromatik** [comment]
Je n'ai aucune idée comment rentrer le kernel également.. 
Sur manjaro. Fr on m'a dit d'essayer ça aussi et j'ai eu ça :

sudo mhwd-kernel -i linux414

[code] [ps4@ps4 ~]$ sudo mhwd-kernel -i linux414
[sudo] Mot de passe de ps4 : 
:: Synchronisation des bases de données de paquets...
 core                     143,6 KiB   335K/s 00:00 [######################] 100%
 extra                   1714,7 KiB   424K/s 00:04 [######################] 100%
 community                  4,5 MiB   425K/s 00:11 [######################] 100%
 multilib                 176,8 KiB   433K/s 00:00 [######################] 100%
résolution des dépendances...
recherche des conflits entre paquets...

Paquets (3) linux414-4.14.29-1  linux414-headers-4.14.29-1
            linux414-virtualbox-guest-modules-5.2.8-6

Taille totale du téléchargement :   69,16 MiB
Taille totale installée :         161,87 MiB

:: Procéder à l’installation ? [O/n] o
:: Récupération des paquets...
 linux414-4.14.29-1-...    60,8 MiB   424K/s 02:27 [######################] 100%
 linux414-headers-4....     8,2 MiB   422K/s 00:20 [######################] 100%
 linux414-virtualbox...   185,5 KiB   390K/s 00:00 [######################] 100%
(3/3) vérification des clés dans le trousseau      [######################] 100%
(3/3) vérification de l’intégrité des paquets      [######################] 100%
(3/3) chargement des fichiers des paquets          [######################] 100%
(3/3) analyse des conflits entre fichiers          [######################] 100%
(3/3) vérification de l’espace disque disponible   [######################] 100%
:: Traitement des changements du paquet...
(1/3) installation de linux414                     [######################] 100%
Dépendances optionnelles pour linux414
    crda: to set the correct wireless channels of your country [installé]
(2/3) installation de linux414-headers             [######################] 100%
(3/3) installation de linux414-virtualbox-gues...  [######################] 100%
===> You may want to load vboxguest, vboxsf and vboxvideo
y:: Exécution des crochets de post-transaction...
(1/4) Updating linux414 module dependencies...

(2/4) Install DKMS modules
==> dkms install vboxhost/5.2.8_OSE -k 4.14.29-1-MANJARO
y
==> dkms install vboxguest/5.2.8_OSE -k 4.14.29-1-MANJARO
(3/4) Updating linux414 initcpios...
==> Building image from preset: /etc/mkinitcpio.d/linux414.preset: 'default'
  -> -k /boot/vmlinuz-4.14-x86_64 -c /etc/mkinitcpio.conf -g /boot/initramfs-4.14-x86_64.img
==> Starting build: 4.14.29-1-MANJARO
  -> Running build hook: [base]
  -> Running build hook: [udev]
  -> Running build hook: [autodetect]
  -> Running build hook: [modconf]
  -> Running build hook: [block]
  -> Running build hook: [keyboard]
  -> Running build hook: [keymap]
  -> Running build hook: [filesystems]
  -> Running build hook: [fsck]
==> Generating module dependencies
==> Creating gzip-compressed initcpio image: /boot/initramfs-4.14-x86_64.img
==> Image generation successful
==> Building image from preset: /etc/mkinitcpio.d/linux414.preset: 'fallback'
  -> -k /boot/vmlinuz-4.14-x86_64 -c /etc/mkinitcpio.conf -g /boot/initramfs-4.14-x86_64-fallback.img -S autodetect
==> Starting build: 4.14.29-1-MANJARO
  -> Running build hook: [base]
  -> Running build hook: [udev]
  -> Running build hook: [modconf]
  -> Running build hook: [block]
  -> Running build hook: [keyboard]
  -> Running build hook: [keymap]
  -> Running build hook: [filesystems]
  -> Running build hook: [fsck]
==> Generating module dependencies
==> Creating gzip-compressed initcpio image: /boot/initramfs-4.14-x86_64-fallback.img
==> Image generation successful
(4/4) Arming ConditionNeedsUpdate...[code]

**Comment by Tikilou** [comment]
Hello, je me permet d'intervenir, étant donné que je connais assez bien Manjaro. ;)

La manipulation que tu as effectuée ici est propre aux PC uniquement, en fait quand une distribution linux est démarrée, il va se passer globalement :+1: 
Boot de l'UEFI (le bios de la machine) => Bootloader (gestionnaire de démarrage GRUB généralement) => Chargement du noyau Linux (bzimage/vmlinuz) qui va piocher une micro-distribution dans l'initramfs contenant des modules (support de matériel, systèmes de fichiers, etc...) qui elle même, va charger une plus grande distribution (Manjaro, Ubuntu, Debian, Fedora...).

Sur PS4, il n'y a pas d'UEFI, la console utilise un firmware maison et un bootloader maison, qui charge un noyau BSD customisé par Sony et OrbitOS basé sur FreeBSD, ensuite via le hack 4.55, le payloader va charger le noyau Linux en mémoire, mais il ne pourra le faire que depuis un système de fichier supporté par OrbitOS et le noyau BSD customisé par Sony, soit, le système de fichier interne, ou l'ExFAT et le Fat16/32.
Il faut savoir que sous Linux, il n'y a pas de support de l'ExFat pour charger l'initramfs, cette solution est donc caduque (car l'exfat est soumis à licence, il y a une histoire de brevets logiciels vis à vis de microsoft derrière), pour charger l'initramfs il faudra donc compter sur une partition fat16/32 ou ext2/3/4 principalement.
Le problème ici, c'est que le seul socle commun entre la PS4 et Linux, c'est le fat16/32, c'est pourquoi le noyau est chargé depuis cette partition, noyau qui, une fois chargé en mémoire par le payload, puis executé, ira chercher l'initramfs sur la même partition fat16/32.

Du coup, si tu utilise la gestion classique d'installation de noyau, quelle que soit la distribution Linux, dédiée aux PC, qui va installer ces derniers dans le répertoire /boot de la partition système Linux formatée en Ext4, le Payload ne le chargera pas, vu qu'il ne peut charger que le noyau présent sur la première partition en fat16/32.

Par ailleurs, il serait bon ne pas oublier que les patchs ps4 pour le noyau Linux, les drivers, ne sont pas encore stables, et n'ont pas été intégré aux versions officielles du noyau Linux, ceux ci sont donc absent de toutes les versions que l'ont installerait de façon classique, donc même si on pouvait booter sur le noyau Linux présent sur la partition système depuis le Payloader de Valentin, on n'aurait aucun support du bridge Panasonic ni des autres éléments dont Fail0verflow a développé le support.

Peut être qu'à terme, quelqu'un ou Valentin sauront comment développer de nouveaux modules pour le noyau (kernel) BSD de la PS4 afin de support plus de systèmes de fichiers et booter directement sur GRUB installé dans /boot sur une partition Ext4, ou exploiter un UEFI maison afin d'utiliser la PS4 comme un PC standard et booter sur le gestionnaire de démarrage GRUB, mais ça n'est actuellement pas le cas, et ça ne le sera peut être jamais.

Donc dans les faits, il est totalement inutile d'utiliser n'importe quel outil standard intégré aux distributions Linux de gestion du noyau Linux. IL y a obligation de faire ça manuelle et de remplacer le bzimage de la partition Fat16/32.

Par ailleurs, ce qui serait vraiment utile, en plus de cette installation manuelle, serait de créer un outil permettant d'installer les kernel-headers de ce noyau Linux copié sur la partition Fat32 et même, de gérer des mises à jour du noyau sur cette dernière.

C'est aussi pour ça que j'ai poussé Manjaro sur PS4, les dépôts AUR, communautaires, et accessibles à tous, permettent facilement de scripter de nouveaux outils disponibles en ligne, et pouvant être dédiés à la PS4.

**Comment by shim80** [comment]
@Tikilou , l'erreur de Astromatik réside principalement dans le fait d'avoir installé le kernel Linux en plus de l'avoir compilé. Car une fois la compilation finie, il a dû procéder à l'installation de celui-ci de façon automatique.

Les fichiers contenus ici: https://github.com/Ps3itaTeam/ps4-linux contiennent le code source d'un kernel Linux spécialement conçu pour tourner sur PS4. Les travaux de la team PSXITA.ITA sous tout récents et visent à concevoir un noyau Linux stable et universel.

Pour cela, ils ont récupéré les travaux de Fail0verflow concernant les pilotes gpu et Eeply pour la prise en charge du MN864729. De plus ils ont rajouté les pilotes du son et de la puce wifi/bt.
Bref. Leur travaux sont encore en cours de développement, mais on a tous très hâte de l'essayer. C'est pourquoi on essaye tant bien que mal de compiler nous même le fichier bzimage depuis leur GitHub afin de tester leur avancées.

**Comment by Astromatik** [comment]
J'ai fait à la barbare quois du Astromatik tout craché ^^ 
Moi qui ne connais absolument rien à Linux,je tâtonne. 
Merci pour les explications tikilou mais comment faire pour compiler tout se sac !!!

**Comment by Tikilou** [comment]
@shim80 
Effectivement, et en l’occurrence, ça ne m_ne donc à rien.
Pour la team PSXITA j'avais vu ça en effet, j'ai hâte aussi, toutefois une bonne documentation de leur part ne serait pas de refus. Espérons également qu'ils pousseront tous leurs apports en commits pour la prochaine version du noyau Linux.

@Astromatik 
On a tous commencé de rien, ne t'inquiète pas, tu apprends ! :)
Pour compiler tout ce "sac", je n'en sais moi même rien, il faudrait que demander à la team PSXITA comment faire, et comment générer le bzimage, s'ils n'ont encore rien publié.

Le libre c'est bien, mais la documentation a aussi son importance, rien de pire que de se retrouver démuni face à un code source dont on ne sait que faire ni comment l'empiler ou l'empaqueter.

**Comment by shim80** [comment]
@Tikilou , à propos, bravo encore d'avoir choisi Manjaro Linux comme distribution dans ton tuto pour la PS4.
J'ai bien vu que ça n'a pas plu aux gars de psxhax (ils n'ont pas cessé de râler sur le pourquoi du comment n'a-t-il pas simplement mis Fedora ou Ubuntu...), mais je te soutiens à 100%.
Étant un grand fan d'Arch Linux, je ne peux qu'approuver ta démarche de répandre davantage l'OS, et proposer AUR aux devs PS4. 
Je n'ai jamais utilisé Manjaro. Je préfère de loins partir de zéro lorsque l'installe Arch Linux. Ça me donne le sentiment d'avoir les pleins pouvoirs...

**Comment by Tikilou** [comment]
@shim80 
Merci, si j'ai choisis Manjaro, c'est justement parce qu'elle a une base modulaire et très accessible, et qu'elle est très proche d'ArchLinux, bien plus que ne le sont Ubuntu et Debian.

Ainsi les débutants peuvent profiter d'une distribution très user-friendly, et les utilisateurs avancés eux, n'auront qu'à installer eux même ArchLinux.

Les gars de pshax se sont débrouillés comme des grands pour installer ubuntu de leur côté, mais comme j'ai donné la primeur avec Manjaro et un article complet, je pense que la plupart des gens vont se concentrer dessus et, juste changer le kernel de la partition FAT pour le support de leur console.

Je ne comptes d'ailleurs pas en rester là, j'attends impatiemment que la team italienne publie un noyau stable et fonctionnel sur tous les aspects.

**Comment by Astromatik** [comment]
Si vous pensez que archlinux est mieux alors, Jesus,  I Rolling with you ¡¡¡
J ai un recule très limité avec Ubuntu ( le seul truc qui m'intéresse sur Ubuntu c'est la possibilité apparemment d' émuler Android ). 
Mais bon... J suis tomber sur ton manjaro tikilou et je le trouve super Class. 
Donc en somme c'est le bzimage qui doit changer si je comprends bien ?

**Comment by shim80** [comment]
Tout à fait, @Astromatik .
Le bzimage contient le noyau compilé. C'est tout ce qui nous manque.

**Comment by Tikilou** [comment]
@Astromatik , on peut aussi facilement émuler Android sous Manjaro/ArchLinux. :)

@shim80 , ce qu'il nous manque, c'est de savoir comment créer nous même ce bzimage de A à Z depuis les sources du noyau Linux.

**Comment by Astromatik** [comment]
@shim80 ah OK bonne nouvelle XD

@tikilou mais alors ce qui montre psita sur la photo en commande, c'est juste la liste de périphériques ? 
Que d'aventure Linux sur PS4, bien + que sur PS3. 
On sent que sa va arraché la moquette pour dire l'expression ^^

**Comment by Tikilou** [comment]
Oui, c'est juste la liste des périphériques, rien de plus... C'est le résultat de la commande "lspci"

Et effectivement, Linux sur PS4 a beaucoup de potentiel, plus qu'il n'en a jamais eu sur console jusque là.

**Comment by Astromatik** [comment]
@Tikilou 
"Linux a le potentiel de nous amener beaucoup plus facilement du homebrew sur PS4, qu'elle ne le permet d'origine avec son SDK ou la nécessité d'écrire un SDK libre, GNU/Linux contient déjà tous les éléments nécessaire pour lancer du homebrews, on peut se baser facilement sur du code déjà existant et créer une belle interface intégrée pour la Ps4 par exemple. (Là encore, les dépôts communautaires pourraient aussi s'avérer être très pratiques.)"

C'est vrai qu'on pourrait imaginer  faire un portage de Retroarch par la suite ou même steam en mode big screen !!! 
Retroarch PS4 (avec Linux) versus Switch, 
Ça va être méchant!!!
On peu même imaginer Rosegarden? Ça serai complètement fou XD La PS4 brancher avec une carte son et un clavier maître... Le Kiff Absolue  !!!!

**Comment by Tikilou** [comment]
@Astromatik , tu as tout compris... Tout est possible... Et des devs console pourraient se pencher sur ce qui a généralement fait le plus défaut pour Linux sur PC... L'interface graphique ! ;)

Plutôt que créer une interface graphique console via un SDK pour OrbisOS, il suffirait de le développer sous Linux... On Parle donc là de l'union de l'univers PC & console, là ou avant sur console, chaque développeur devait galérer quasiment tout seul sur son homebrew... Chaque univers pourra contribuer à l'autre.

L'idéal, serait de trouver une voie pour pouvoir lancer Linux depuis un homebrew installé sous forme de PKG sur la PS4, et lancer Linux dans une Sandbox, comme tous les autres jeux/programmes, tout en faisant sauter les limitations des sandbox pour que Linux puisse exploiter tout le matos comme si c'était natif. On pourrait faire des tas de microdistributions Linux avec des émulateurs, etc, à lancer directement depuis le menu PS4

**Comment by Astromatik** [comment]
@shim80, @Tikilou et @valentinbreiz. 
Pensez vous qu'il serait judicieux de parler aussi de ça sur wololo. net ?

**Comment by jja2000** [comment]
I'm sorry for being late to the party, but trying to boot a kernel on my CUH-1216 has been kinda hard.

Installing the Manjaro build from psxhax only gives me a white flashing LED (so it booted), but no ssh or display. Installing eeply's newest kernel release gives me the same outcome but with ssh. The dmesg points out that there's a problem:
`[  162.650632] [drm:radeon_dp_link_train] *ERROR* clock recovery tried 5 times
[  162.650635] [drm:radeon_dp_link_train] *ERROR* clock recovery failed
[  162.966609] [drm:radeon_dp_link_train] *ERROR* clock recovery tried 5 times
[  162.966613] [drm:radeon_dp_link_train] *ERROR* clock recovery failed
`
I'm guessing it has to do with the HDMI converter, but your guess is as good as mine.
My PS4 is connected to my monitor an IIyama ProLite E2473HS, it's a 1080p 60Hz monitor.

**Comment by Astromatik** [comment]
HUM no when u boot on linux your led is bleu and pink on the down.
Maybe you can try other init or bzimage.
IF you are on my model cuh1216A, try to change resolution .

**Comment by jja2000** [comment]
The led colour is the proper one haha, it's just described differently per website.

Got any good init and bzimage suggestions? I tried to use eeply's but it didn't work.

Changing resolutions you can do by recompiling ps4-linux-loader with the proper cmdline right?

**Comment by Astromatik** [comment]
http://www.logic-sunrise.com/forums/topic/80934-ps4-un-correctif-linux-pour-les-modeles-cuh-12xx/?fromsearch=1

GOOGLE TRAD XD

**Comment by jja2000** [comment]
I have a full hd screen though. 720p won't make any difference...

**Comment by zoenders** [comment]
有人会中文吗？这个PS4 Linux 5.05怎么用，能教教我吗？

**Comment by valentinbreiz** [comment]
对不起，我不会说中文，但是对于5.05，你需要一个内核缺陷，这个缺陷尚未被开发者发布！

**Comment by ghost** [comment]
I have the same problem with the PS4 having a panic and shutting down after sending the payload, it seems as though the screen goes black as though its going to start to load Linux and maybe go to a rescue shell but it turns out the model (CUH-1215A) I am trying to install Linux to isn't supported currently by the payload

CUH-1115B working
CUH-1102A working 
CUH-1115A working 
CUH-1116A working
CUH-1004A working
CUH-1001A working
CUH-1116B working



CUH-1215A not working
CUH-1216A not working 
CUH-1208B not working 
CUH-1216 not working 
CUH-1216B not working 
CUH-2015A not working
CUH-1008A not working
CUH-7016B not working
CUH-7015B not working
CUH-2016B not working




please see here
https://twitter.com/valentinbreiz/status/947864073085079552?lang=en

**Comment by ghost** [comment]
I tried to experiment with CUH-1215A, I recompiled to 720p no big deal I guess


I just recompiled a 5.05 Linux Loader based on this code, but for 720p
because I was just experiencing the same problem in the last few days trying to load Fedora,

^ this is for the Linux Loader for 5.05 in 720p for CUH-1215A models, probably most CUH-12XX models,
you have to switch your PS4 video resolution from 1080p to 720p
its working for me, using the same initrd and bzImage for 5.05

PS4-Linux-Loader-5.05.720p.bin
https://mega.nz/#!P3wEWCLL!fGI_LSwbNo7qwW7X_didKKs59XTp_6qkVnkF79eSIzo

bzImage and initrd

5.05-fat32-files-for-720p-1080p-loader.zip
https://mega.nz/#!Sixg1aBD!HDtq6qDA8NR-Ta3Rpukt1mB6GNx-euTQnXUipG3YH70

**Comment by ghost** [comment]
this was a similar problem on the ps3 with petitboot and 720p resolution in the beginning

**Comment by DragonLord7791** [comment]
linux not working for my ps4 cuh 2015
black screan and solid white led
i did tested on panasonic viera 2009 and Hisense 2018 hdmi 1080p both
also i did try whit 720p,same resoult

**Comment by mirh** [comment]
Possible solutions:
https://twitter.com/kozarovv/status/1096732559063633921
https://twitter.com/il__rib/status/1096864563012747264

**Comment by jja2000** [comment]
> Possible solutions:
> https://twitter.com/kozarovv/status/1096732559063633921
> https://twitter.com/il__rib/status/1096864563012747264

I have not passed the rescueshell yet with the new release sadly
The new initramfs wouldn't/doesn't work

**Comment by TheAncientGreek** [comment]
Pleaseeeeeee make a release for cuh 2016a. I got black screen eith no signal :((((

**Comment by mirh** [comment]
Try the initramfs [here](http://www.logic-sunrise.com/forums/topic/85999-ps4-un-fix-pour-les-ecrans-est-disponible-pour-linux/) (rename it on the usb drive btw). 
(*which I think.. should be [this](https://github.com/Ps3itaTeam/PS4-Linux-Loader/tree/5.05) compiled?*)

**Comment by tonyyoyo** [comment]
Really, guys? You're going to let this go on for years?! Get up off your a**, and fix it!

**Comment by tonyyoyo** [comment]
Resolved [here](https://www.psxhax.com/threads/baikal-support-added-to-playstation-4-linux-loader-by-valeery.7165/).
